server/views/pages/overview/partials/overviewTabActionPlanColumn.njk

```diff
@@ -85,7 +85,14 @@
         </table>
 
         {% if goal.note %}
-          <p class="govuk-body">Note: {{ goal.note }}</p>
+          {{ govukDetails({
+            summaryHtml: "View note<span class='govuk-visually-hidden'> for Goal " + loop.index + "</span>",
+            text: goal.note,
+            classes: "govuk-!-display-none-print",
```

Comment:
Pre-empting the work we're doing on print and using the govuk utility class to hide on print.